src/types.ts

```diff
@@ -8,6 +8,10 @@ export interface Env {
 	ENABLE_REAL_THINKING?: string; // Optional flag to enable real Gemini thinking output (set to "true" to enable)
 	STREAM_THINKING_AS_CONTENT?: string; // Optional flag to stream thinking as content with <thinking> tags (set to "true" to enable)
 	ENABLE_AUTO_MODEL_SWITCHING?: string; // Optional flag to enable automatic fallback from pro to flash on 429 errors (set to "true" to enable)
+	GEMINI_MODERATION_HARASSMENT_THRESHOLD?: string;
```

Comment:
Can we make this a type instead of string?
so
```
export type SafetyThreshold = 'BLOCK_NONE' | 'BLOCK_FEW' | 'BLOCK_SOME' | 'BLOCK_ONLY_HIGH' | 'HARM_BLOCK_THRESHOLD_UNSPECIFIED'

GEMINI_MODERATION_HARASSMENT_THRESHOLD?: SafetyThreshold ;
```